tests/languages/java-file/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM openjdk:8-alpine
```

Comment:
Please change file structure as java / java_file.

Reply:
Done. Created java_hello_world and java_file under java folder. Will add new java-* folders under here